kernel/inferCumulativity.ml

```diff
@@ -259,23 +259,20 @@ let infer_arity_constructor is_arity env variances arcn =
      i is irrelevant, j is invariant. *)
   if not is_arity then infer_term CUMUL env variances codom else variances
 
-open Entries
-
-let infer_inductive_core env univs entries =
+let infer_inductive_core ~env_params ~env_ar_par ~arities ~ctors univs =
   let variances = Inf.start univs in
-  let variances = List.fold_left (fun variances entry ->
-      let variances = infer_arity_constructor true
-          env variances entry.mind_entry_arity
-      in
-      List.fold_left (infer_arity_constructor false env)
```

Comment:
We used to run the inference for constructors in an env with the arities as unbound rels, I don't think it caused issues ut I fixed it regardless.